src/app/task/task.controller.ts

```diff
@@ -0,0 +1,62 @@
+import {
+  Body,
+  Controller,
+  Delete,
+  Get,
+  HttpCode,
+  Param,
+  ParseIntPipe,
+  Post,
+  Put,
+  Query,
+} from '@nestjs/common';
+import { TaskService } from './task.service';
+import { Task } from 'src/entities/task.entity';
+import { ApiBearerAuth, ApiQuery, ApiTags } from '@nestjs/swagger';
+import { TaskDto } from './dto/task.dto';
+import { GetUser } from 'src/decorator/get-user.decorator';
+import { User } from 'src/entities/user.entity';
+
+@ApiTags('Task')
+@ApiBearerAuth()
```

Comment:
you should use jwt guard if you want to @GetUser to work

Reply:
done